### PR TITLE
Merge qa to master: update feature version

### PR DESCRIPTION
**Pull request by gkwan-ibm**
#65 #67

### CODE REVIEW

File: finish/system/pom.xml [hunk 1]
```diff
@@ -35,12 +35,12 @@
         </dependency>
         <dependency>
             <groupId>io.openliberty.features</groupId>
-            <artifactId>mpRestClient-1.1</artifactId>
```
Comment (proubatsis) [inline finish/system/pom.xml]: The `mpRestClient` feature is not specified in the `server.xml` file. Is it being used?

File: start/system/pom.xml [hunk 2]
```diff
@@ -35,12 +35,12 @@
         </dependency>
         <dependency>
             <groupId>io.openliberty.features</groupId>
-            <artifactId>mpRestClient-1.1</artifactId>
```
Comment (proubatsis) [inline start/system/pom.xml]: Same comment but for the `start/` directory.

File: finish/system/pom.xml (hunk 1 above)
Comment (gkwan-ibm) [reply finish/system/pom.xml]: The test may require it

File: start/system/pom.xml (hunk 2 above)
Comment (gkwan-ibm) [reply start/system/pom.xml]: The test may require it

File: finish/system/pom.xml (hunk 1 above)
Comment (gkwan-ibm) [reply finish/system/pom.xml]: addressed by #67

File: start/system/pom.xml (hunk 2 above)
Comment (gkwan-ibm) [reply start/system/pom.xml]: addressed by #67

**Pull request merged**